arwen/host/execution.go

```diff
@@ -710,6 +726,9 @@ func (host *vmHost) callSCMethod() error {
 	if err != nil {
 		err = host.handleBreakpointIfAny(err)
```

Comment:
If function() returns err this call could overwrite the init error with nil, it is ok?

Reply:
I would leave like this. It is the correct way to do it.

Reply:
The code is correct